SparkPost/2web2ui
# TR-2003 - Resolve issue with requests not working when subaccount ID is required

[TR-2003](https://jira.int.messagesystems.com/browse/TR-2003)

### What Changed
- Updated calls to `getDraft` and `getPublished` to pass template `subaccount_id`

### How To Test
1. Create a new template and assign it to a subaccount
2. On the templates list page, click on the duplicate button in both the recent activity section and the table
3. Verify that duplicating the template succeeds in both instances

### To Do
